src/leds.c

```diff
@@ -261,6 +275,40 @@ static void anim_knight_rider(Leds *leds, const LedStrip *strip, const LedBar *b
     }
 }
 
+static void anim_felony(Leds *leds, const LedStrip *strip, const LedBar *bar, float time) {
+    uint32_t color_off = 0x00000000;
+
+    float state_duration = 0.05f;
+    float state_mod = fmodf(time, 3.0f * state_duration);
+
+    // also account for led strips with odd numbers of leds (leaving the middle one black)
+    uint8_t stop_idx = strip->length / 2;
+    uint8_t start_idx = strip->length / 2 + (strip->length % 2 * 2);
```

Comment:
I think the `* 2` is wrong, it'll result in two black LEDs unless I'm mistaken.

Reply:
Ahh, the 3rd hardest of the 2 hardest things in programming :man_facepalming: fixed!